_data/en/settings.yml

```diff
@@ -60,6 +60,7 @@ contact_page:
       'Housing and Urban Development': Housing and Urban Development
       'Internal Revenue Service (IRS)': Internal Revenue Service (IRS)
       'Library of Congress': Library of Congress
+      'National Archives and Records Administration (NARA)': National Archives and Records Administration (NARA)
```

Comment:
Minor: Alphabetically, this would come after "National Aeronautics and Space Administration". Looks like the page has code to alphabetize it pragmatically though.